test/fixtures/dialects/spark3/insert_overwrite_directory_hive.sql

```diff
@@ -0,0 +1,7 @@
+INSERT OVERWRITE LOCAL DIRECTORY '/tmp/destination'
+STORED AS ORC
+SELECT * FROM test_table;
+
+INSERT OVERWRITE LOCAL DIRECTORY '/tmp/destination'
+ROW FORMAT DELIMITED FIELDS TERMINATED BY ','
+SELECT * FROM test_table;
```

Comment:
Any chance of a few more test cases to cover some of the optional parts above?

Reply:
Actually I missed that these are already covered by existing test cases. OK good to merge.